src/Day.jsx

```diff
@@ -46,13 +43,12 @@ export default class Day extends PureComponent {
 
   relativeY(pageY, rounding = ROUND_TO_NEAREST_MINS) {
     const { top } = this.mouseTargetRef.getBoundingClientRect();
-    let realY = pageY - top - document.body.scrollTop;
+    let realY = pageY - top - document.documentElement.scrollTop;
```

Comment:
@trotzig  sorry, This is my first PR on github, Because I used Prettier to format the code , SO I didn't submit it clearly 。 The problem is in  here , `let realY = pageY - top - document.documentElement.scrollTop;` I replaced `document.body.scrollTop`  with `document.documentElement.scrollTop`  , When I use `document.body.scrollTop`  is I can not get rolling height，Is this problem encountered or solved?

Reply:
Congrats on making your first pull-request! I'm honored that you did it in my project!

I haven't encountered this issue before, but I think I haven't been using this component on pages with scroll. Glad you found it!